pkg/cmd/cli/cmd/status.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/origin/pkg/cmd/cli/describe"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+const statusLongDesc = `
+Show a high level overview of the current project. Links components by their relationships.
+For more information about individual items, use the describe command (e.g. osc describe buildConfig,
+osc describe deploymentConfig, osc describe service).
+`
+
+func NewCmdStatus(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "status",
+		Short: "Show an overview of the current project",
+		Long:  fmt.Sprintf(statusLongDesc, fullName),
+		Run: func(cmd *cobra.Command, args []string) {
+			client, kclient, err := f.Clients()
+			checkErr(err)
+
+			namespace, err := f.DefaultNamespace()
+			checkErr(err)
+
+			describer := &describe.ProjectStatusDescriber{kclient, client}
+			s, err := describer.Describe(namespace, "")
+			checkErr(err)
+
+			fmt.Fprintf(out, s)
+		},
+	}
+	return cmd
```

Comment:
Naw, I don't think we'd do that either.  Let's just say that we will evolve status in the future.  I don't think watch will work from a CLI experience really.

----- Original Message -----
> > +		Long:  fmt.Sprintf(statusLongDesc, fullName),
> > +		Run: func(cmd *cobra.Command, args []string) {
> > +			client, kclient, err := f.Clients()
> > +			checkErr(err)
> > +
> > +			namespace, err := f.DefaultNamespace()
> > +			checkErr(err)
> > +
> > +			describer := &describe.ProjectStatusDescriber{kclient, client}
> > +			s, err := describer.Describe(namespace, "")
> > +			checkErr(err)
> > +
> > +			fmt.Fprintf(out, s)
> > +		},
> > +	}
> > +	return cmd
> 
> Add a TODO to support `--watch` (may be hard).
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1314/files#r27505138